helm/templates/configmap.yaml

```diff
@@ -62,5 +64,6 @@ data:
   {{ if $metricsEnabled }}
   TELEMETRY_METRICS_URL: {{ $metricsUrl }}
   TELEMETRY_METRICS_BUCKETS: {{ $metricsBuckets | toJson | quote }}
+  MAPPROXY_CACHE_GRIDS: {{ .Values.env.mapproxyCache.grids | quote }}
```

Comment:
please keep in the same order as in config/custom-environment-variables.json

Reply:
done